src/components/Table/Table.mocks.tsx

```diff
@@ -0,0 +1,250 @@
+/**
```

Comment:
This moks file are excluded from final bundle from tsconfig configuration?
```
"exclude": [
    ..., "**/*.mocks.ts"
  ]
  ```

Reply:
We can do this in an ad hoc pr

Reply:
let's create an issue regarding bundle size